README.md

```diff
@@ -19,7 +19,7 @@ Requirements for installing AppStudio in E2E mode and running the E2E tests:
 
 * An OpenShift 4.11 or higher Environment (If you are using CRC/OpenShift Local please also review [optional-codeready-containers-post-bootstrap-configuration](https://github.com/redhat-appstudio/infra-deployments#optional-codeready-containers-post-bootstrap-configuration))
 * A machine from which to run the install (usually your laptop) with required tools:
-  * A properly setup Go workspace using **Go 1.18 is required**
+  * A properly setup Go workspace using **Go 1.19 is required**
```

Comment:
[go.mod](https://github.com/redhat-appstudio/e2e-tests/pull/406/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6) still specifies go v1.18

if we are about to move to go v1.19, we should probably bump the base image version [here](https://github.com/redhat-appstudio/e2e-tests/blob/5c66b55767fc11ba2f1172896f98921bcdd329a5/Dockerfile#L1) and [here](https://github.com/openshift/release/blob/c233758d9bd68641e11f5922f66da9f1c1ce9a2e/ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml#L5) as well

Reply:
This was not intentional. While I was trying to debug I did update to 1.19 to see if the error go away, but it didn't. I had forgotten to revert this line. Thanks for catching it!